.gitmodules

```
@@ -0,0 +1,3 @@
+[submodule "extensions"]
```

I guess it might be better to call them _external_ to make it more separated from the rest.

```suggestion
[submodule "external/extensions"]
```

We might also need to modify extensions dir path:
```env
EXTENSIONS_DIR_PATH=/workspace/external/extensions
```

---

They a replacing the folder. There will be no extension code in the the repo any more. @safeamiiir  May you please motivate this change?

---

Yeah I get that.
It's more about putting `extensions` inside `external` directory, just to emphasise it's external and if we have more submodules in any future, it also end up in the same external directory.